.eslintrc.js

```diff
@@ -24,7 +27,7 @@ module.exports = {
     "no-plusplus": "off",
     "no-proto": "error",
     "no-return-await": "error",
-    "no-underscore-dangle": "error",
+    "no-underscore-dangle": "off",
```

Comment:
Any reason for this? Just wondering since this is repository-wide configuration, and we decided on these defaults as a group.

Reply:
we name private variables and private methods with _

Reply:
OK, sounds good.